values.yaml

```diff
@@ -5,6 +5,11 @@ image:
 #
 deployment:
   replicas: 1
+
+# Additional arguments to be passed at Traefik's binary
+additionalArguments: []
```

Comment:
```suggestion
## Use curly braces to pass values: `helm install --set="{--providers.kubernetesingress,--log.level=DEBUG}" ."
additionalArguments: []
```

Reply:
That's a good point. I never use --set={}, I always copy the values.yml file and edit it

Reply:
I wasn't aware of the commit suggestion feature of GH, that's nice.